.github/workflows/build-engines.yml

```diff
@@ -69,6 +69,7 @@ jobs:
           (
             github.event.pull_request.author_association == 'OWNER' ||
             github.event.pull_request.author_association == 'MEMBER' ||
+            github.event.pull_request.author_association == 'CONTRIBUTOR' ||
```

Comment:
why is this change necessary in this PR?

Reply:
Yes, otherwise the Linux Build step would've been skipped. My user role is "CONTRIBUTOR", not "COLLABORATOR", according to the action's logs